src/OrchardCore/OrchardCore.ContentManagement.Display/ContentDisplayManager.cs

```diff
@@ -169,11 +169,11 @@ ILayoutAccessor layoutAccessor
 
             var updateContentContext = new UpdateContentContext(contentItem, updater);
 
-            _contentHandlers.Invoke(handler => handler.Updating(updateContentContext), Logger);
+            _contentHandlers.Invoke(handler => handler.UpdatingAsync(updateContentContext), Logger);
```

Comment:
Without looking at the code, I think there is an async Invoke. And this should be awaited.

Reply:
You're right, I'll fix.